index.html

```diff
@@ -11,15 +11,18 @@
   <body>
     <header class="header">
       <a href="#" class="logo-nombre">Afiamoah Samuel</a>
-      <div class="hamburger">
-        <nav class="navlinks">
-          <ul>
-            <li><a href="#porfolio">Portfolio</a></li>
-            <li><a href="#about">About</a></li>
-            <li><a href="#contact">Contact</a></li>
-          </ul>
-        </nav>
+      <nav class="navlinks">
+        <ul class="nav-container">
+          <li class="nav-items"><a href="#portfolio">Portfolio</a></li>
+          <li class="nav-items"><a href="#about">About</a></li>
+          <li class="nav-items"><a href="#contact">Contact</a></li>
+        </ul>
+           <div class="hamburger">
+        <span class="ham"></span>
+        <span class="ham"></span>
+        <span class="ham"></span>
       </div>
+      </nav>
```

Comment:
-  [ ] Good job you implemented the menu :+1:  However there is some issue you need to address with your menu design. :cry:   It would be nice if you add the black bar at the end of your menu and the small grey spacing to follow the Figma template :+1: Please fix it in order to have the similar to the Figma template :smile: 

You can check the screenshot below :point_down: 

#### Your mobile menu
![Screenshot from 2022-07-26 19-10-57](https://user-images.githubusercontent.com/6615394/181095041-91b785ec-66c8-4af1-90e6-458baa3eadf0.png)

#### Figma template
![Screenshot from 2022-07-26 19-11-18](https://user-images.githubusercontent.com/6615394/181095075-448ce4c0-3000-4ff7-98bf-dad85d34152b.jpg)


- [ ] Please use the same color `#FFF5E1` for your menu mobile option and ensure you have the same color and size for your close button as defined on Figma template to make them similar :+1:

Reply:
menu design issues fixed.now similar to figma design